spec/models/comment_spec.rb

```diff
@@ -0,0 +1,14 @@
+require 'rails_helper'
+
+RSpec.describe Comment, type: :model do
+  let(:user) { User.create(name: 'John Doe', photo: 'photo', bio: 'bio') }
+  let(:post) { Post.create(title: 'title', text: 'text', author: user) }
+
+  let(:comment) { Comment.new(text: 'text', user_id: user.id, post_id: post.id) }
+
+  before { comment.save }
+
+  it 'is valid with valid attributes' do
+    expect(comment).to be_valid
+  end
```

Comment:
- [ ] I think it would be great if you test the method update comments counter as well since you should test all methods in the models so kindly test it 💯💪🏻👍🏻

![image](https://github.com/Lawmsangi/blog-app/assets/58553711/e404dee2-10a2-4555-a5d1-877c9ca73af1)